src/java/com/puppetlabs/http/client/impl/PersistentAsyncHttpClient.java

```diff
@@ -15,13 +15,16 @@
     private CloseableHttpAsyncClient client;
     private MetricRegistry metricRegistry;
     private String metricNamespace;
+    private Boolean useURLMetrics;
 
     public PersistentAsyncHttpClient(CloseableHttpAsyncClient client,
                                      MetricRegistry metricRegistry,
-                                     String metricNamespace) {
+                                     String metricNamespace,
+                                     Boolean useURLMetrics) {
```

Comment:
It looks like I've used `boolean` and `java.lang.Boolean` interchangeably. Is there a best practice around that? Will difference in them potentially screw up clojure interop?